ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -57,6 +57,120 @@ presubmits:
           name: prow-job-cluster-launch-installer-e2e
         name: job-definition
     trigger: ((?m)^/test( all| e2e-aws),?(\s+|$))
+  - agent: kubernetes
```

Comment:
why are we adding 2 jobs `e2e-aws-op` and `e2e-aws-operator` ?

Reply:
Fixed